File: 310971/Stankevich/rest310/src/main/java/com/task/rest/model/Author.java
package com.task.rest.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Author {
    private Long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 251003/Golovin/appLab2/src/main/kotlin/com/example/eger/dc/lab2/bean/Author.kt
package com.example.eger.dc.lab2.bean

import com.example.eger.dc.lab2.dto.response.AuthorResponseTo
import kotlinx.serialization.Serializable

@Serializable
data class Author(
	val id: Long?, val login: String, val password: String, val firstname: String, val lastname: String
) {
	fun toResponse(): AuthorResponseTo = AuthorResponseTo(id, login, password, firstname, lastname)
}